react/week4/githubusers/src/UserContext.js

```diff
@@ -0,0 +1,3 @@
+import React from 'react';
+
+export const UserContext = React.createContext({ userId: 10 });
```

Comment:
What is this for? :)

Reply:
The teacher had said that we need user context for authentication , to figure out if the logged user has an id or not,.,but i also think its not needed in this case